ecs_deploy/cli.py

```diff
@@ -364,22 +454,34 @@ def scale(cluster, service, desired_count, access_key_id, secret_access_key, reg
-@click.option('--public-ip', is_flag=True, default=False, help='Should a public IP address be assigned to the task (default: False)')
-@click.option('--platform-version', help='The version of the Fargate platform on which to run the task. Optional, FARGATE launch type only.', required=False)
+@click.option('-c', '--command', type=(str, str), multiple=True,
+              help='Overwrites the command in a container: <container> <command>')
+@click.option('-e', '--env', type=(str, str, str), multiple=True,
+              help='Adds or changes an environment variable: <container> <name> <value>')
+@click.option('--env-file', type=(str, str), default=((None, None),), multiple=True, required=False,
+              help='Load environment variables from .env-file')
+@click.option('-s', '--secret', type=(str, str, str), multiple=True,
+              help='Adds or changes a secret environment variable from the AWS Parameter Store (Not available for Fargate): <container> <name> <parameter name>')
+@click.option('--launchtype', type=click.Choice([LAUNCH_TYPE_EC2, LAUNCH_TYPE_FARGATE]), default=LAUNCH_TYPE_EC2,
+              help='ECS Launch type (default: EC2)')
+@click.option('--subnet', type=str, multiple=True,
+              help='A subnet ID to launch the task within. Required for launch type FARGATE (multiple values possible)')
+@click.option('--securitygroup', type=str, multiple=True,
+              help='A security group ID to launch the task within. Required for launch type FARGATE (multiple values possible)')
+@click.option('--public-ip', is_flag=True, default=False,
+              help='Should a public IP address be assigned to the task (default: False)')
+@click.option('--platform-version',
+              help='The version of the Fargate platform on which to run the task. Optional, FARGATE launch type only.',
+              required=False)
 @click.option('--region', help='AWS region (e.g. eu-central-1)')
 @click.option('--access-key-id', help='AWS access key id')
 @click.option('--secret-access-key', help='AWS secret access key')
 @click.option('--profile', help='AWS configuration profile name')
-@click.option('--exclusive-env', is_flag=True, default=False, help='Set the given environment variables exclusively and remove all other pre-existing env variables from all containers')
+@click.option('--exclusive-env', is_flag=True, default=False,
+              help='Set the given environment variables exclusively and remove all other pre-existing env variables from all containers')
 @click.option('--diff/--no-diff', default=True, help='Print what values were changed in the task definition')
-def run(cluster, task, count, command, env, env_file, secret, launchtype, subnet, securitygroup, public_ip, platform_version, region, access_key_id, secret_access_key, profile, exclusive_env, diff):
+def run(cluster, task, count, command, env, env_file, secret, launchtype, subnet, securitygroup, public_ip,
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Using more than five non-keyword parameters in a function definition can indicate that the function is difficult to understand and maintain. Try to combine parameters in objects or refactor the function into small functions.